lib/pure/times.nim

```diff
@@ -744,6 +747,336 @@ proc format*(info: TimeInfo, f: string): string =
 
 {.pop.}
 
+proc getMonth(m: int): Month = 
+    case m
+    of 1:
+        return mJan
+    of 2:
+        return mFeb
+    of 3:
+        return mMar
+    of 4:
+        return mApr
+    of 5:
+        return mMay
+    of 6:
+        return mJun
+    of 7:
+        return mJul
+    of 8:
+        return mAug
+    of 9:
+        return mSep
+    of 10:
+        return mOct
+    of 11:
+        return mNov
+    of 12:
+        return mDec
+    else:
+        raise newException(ValueError, "invalid month") 
+
+proc parseToken(info: var TimeInfo; token, value: string; j: var int) =
+    ## Helper of the parse proc to parse individual tokens.
+    case token
+    of "d":
+        if value[j+1] in digits:
+            #two digit format
```

Comment:
I know this is picky but please add a space after the ``#`` in your comments, i.e. "# two digit format".